lib/views/forget_password/forget_password_view.dart

```diff
@@ -0,0 +1,146 @@
+        child: Stack(
+          children: [
+            _backgroundImage,
+            _linearGradientBlurBackground,
+          ],
+        ),
+      );
+
+  Widget get _backgroundImage => Image(
+        image: Assets.images.nimbleBackground.image().image,
+        fit: BoxFit.cover,
+        alignment: Alignment.center,
+      );
+
+  Widget get _linearGradientBlurBackground => Stack(
+        children: [
+          _backgroundImage,
+          ImageFiltered(
+            imageFilter: ImageFilter.blur(sigmaX: 10, sigmaY: 10),
+            child: ShaderMask(
+              shaderCallback: (rectangle) {
+                return LinearGradient(
+                  begin: Alignment.topCenter,
+                  end: Alignment.bottomCenter,
+                  colors: [
+                    Colors.black.withOpacity(0.2),
+                    Colors.black.withOpacity(1),
+                  ],
+                ).createShader(rectangle);
+              },
+              blendMode: BlendMode.overlay,
+              child: _backgroundImage,
+            ),
+          ),
+        ],
+      );
+
+  Widget get _logo => SizedBox(
+        width: 166,
+        height: 40,
+        child: FittedBox(
+          fit: BoxFit.contain,
+          child: Image(
+            image: Assets.images.nimbleLogoWhite.image().image,
+            fit: BoxFit.scaleDown,
+          ),
+        ),
+      );
+
+  Widget _intruction(BuildContext context) => Text(
+        'Enter your email to receive instructions for resetting your password.',
+        style: Theme.of(context)
+            .textTheme
+            .bodyMedium
+            ?.copyWith(color: Colors.white70),
+        textAlign: TextAlign.center,
+      );
+
+  Widget get _form => const ForgetPasswordForm();
+
+  @override
+  Widget build(BuildContext context) {
+    _setupStateListener();
+    return _defaultForgetPasswordView();
+  }
+
+  Widget _defaultForgetPasswordView() {
+    return Stack(
+      children: [
+        _background,
+        Scaffold(
+          backgroundColor: Colors.transparent,
+          appBar: _appBar,
+          body: SafeArea(
+            child: Container(
+              padding: const EdgeInsets.symmetric(vertical: 30, horizontal: 24),
+              child: Column(
+                children: [
+                  const SizedBox(height: 60),
+                  _logo,
+                  const SizedBox(height: 20),
+                  _intruction(context),
+                  const SizedBox(height: 100),
+                  _form,
+                ],
+              ),
+            ),
+          ),
+        ),
+      ],
+    );
+  }
+
+  void _setupStateListener() {
+    ref.listen<ForgetPasswordState>(forgetPasswordViewModelProvider,
+        (_, state) {
+      context.showOrHideLoadingIndicator(
+        shouldShow: state == const ForgetPasswordState.loading(),
+      );
+      state.maybeWhen(
+        error: (error) {
+          ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
@nmint8m I think we should put `showSnackBar` in `ContextExt` for convenience using. What do you think?